### PR TITLE
Refactoring ineffassign varialbe 

### PR DESCRIPTION
**Pull request by daehyeok**
Change unnecessary variable name to underbar Remove unnecessary assignment

Signed-off-by: Daehyeok Mun <daehyeok@gmail.com>

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

File: pkg/controller/daemon/daemoncontroller.go [hunk 1]
```diff
@@ -289,7 +289,6 @@ func isDaemonSetMatch(pod *v1.Pod, ds *extensions.DaemonSet) bool {
 	}
 	selector, err := unversioned.LabelSelectorAsSelector(ds.Spec.Selector)
 	if err != nil {
-		err = fmt.Errorf("invalid selector: %v", err)
```
Comment (brendandburns) [inline pkg/controller/daemon/daemoncontroller.go]: Change this to a `glog.Warningf(...)`

File: pkg/controller/replicaset/replica_set.go [hunk 2]
```diff
@@ -272,7 +272,6 @@ func isReplicaSetMatch(pod *v1.Pod, rs *extensions.ReplicaSet) bool {
 	}
 	selector, err := unversioned.LabelSelectorAsSelector(rs.Spec.Selector)
 	if err != nil {
-		err = fmt.Errorf("invalid selector: %v", err)
```
Comment (brendandburns) [inline pkg/controller/replicaset/replica_set.go]: here too.

File: pkg/kubectl/cmd/certificates.go [hunk 3]
```diff
@@ -179,7 +179,7 @@ func (options *CertificateOptions) modifyCertificateCondition(f cmdutil.Factory,
 		}
 		csr := info.Object.(*certificates.CertificateSigningRequest)
 		csr, verb := modify(csr)
-		csr, err = c.Certificates().
```
Comment (brendandburns) [inline pkg/kubectl/cmd/certificates.go]: I don't think that this is correct to remove.

File: pkg/kubectl/describe.go [hunk 4]
```diff
@@ -2699,7 +2699,6 @@ func printLabelsMultilineWithIndent(w *PrefixWriter, initialIndent, title, inner
 			w.Write(LEVEL_0, "%s", innerIndent)
 		}
 		w.Write(LEVEL_0, "%s=%s\n", key, labels[key])
-		i++
```
Comment (brendandburns) [inline pkg/kubectl/describe.go]: Pretty sure this shouldn't be removed.

File: pkg/util/ebtables/ebtables.go [hunk 5]
```diff
@@ -123,14 +123,11 @@ func (runner *runner) GetVersion() (string, error) {
 }
 
 func (runner *runner) EnsureRule(position RulePosition, table Table, chain Chain, args ...string) (bool, error) {
-	exist := true
 	fullArgs := makeFullArgs(table, opListChain, chain, fullMac)
 	out, err := runner.exec.Command(cmdebtables, fullArgs...).CombinedOutput()
-	if err != nil {
```
Comment (brendandburns) [inline pkg/util/ebtables/ebtables.go]: this is definitely not the same as the new code.

**Comment by brendandburns** [comment]
Thanks for the PR!

I noted a number of places where this doesn't look quite right.

File: pkg/util/ebtables/ebtables.go (hunk 5 above)
Comment (daehyeok) [inline pkg/util/ebtables/ebtables.go]: oops i confused. exists same with err.

File: pkg/kubectl/describe.go (hunk 4 above)
Comment (daehyeok) [inline pkg/kubectl/describe.go]: i'm curios, is it really work? anyway for loop works for every elements, and i is not used.
```
func main() {
	inList := []string{"first", "second", "third", "fourth"}
	for i, v := range inList {
		fmt.Println(i, v)
		i++
	}
}
```
output
```
0 first
1 second
2 third
3 fourth
```

Comment (brendandburns) [inline pkg/kubectl/describe.go]: ah, yes, good point. this is ok to remove.

**Comment by bgrant0607** [comment]
Closing due to lack of follow up.

**Pull request closed without merging**